d2bs/kolbot/libs/MuleLogger.js

```diff
@@ -18,16 +19,17 @@ var MuleLogger = {
 			Individual entries are separated with a comma.
 		*/
 
-		"account/password/realm": ["all"]
+		"accountname/password/realm": ["all"]
 	},
 
 	LogGame: ["", ""], // ["gamename", "password"]
-	LogNames: true, // Put account/character name on the picture
+	LogNames: false, // Put account/character name on the picture
 	LogItemLevel: true, // Add item level to the picture
-	LogEquipped: false, // include equipped items
-	LogMerc: false, // include items merc has equipped (if alive)
+	LogEquipped: true, // include equipped items
+	LogMerc: true, // include items merc has equipped (if alive)
 	SaveScreenShot: false, // Save pictures in jpg format (saved in 'Images' folder)
-	IngameTime: 20, // Time to wait after leaving game
+	IngameTime: ((Math.random() * 30) + 180), // Time to wait in game to do not get RD
```

Comment:
leave this as a simple number value, you can add the random time where IngameTime is actually used or use something like rand(30, 180) like you did with the starter config